core/src/script/CGXP/plugins/Print.js

```diff
@@ -345,23 +345,27 @@ cgxp.plugins.Print = Ext.extend(gxp.plugins.Tool, {
             useNativeAngle: true,
             serverType: 'mapserver'
         };
-        this.encodeLayer = this.encodeLayer === undefined || encodeLayer;
-        this.encodeExternalLayer = this.encodeExternalLayer === undefined || encodeLayer;
-
-        this.additionalAttributes = this.additionalAttributes === undefined || this.version == 2 ?
-            [{
-                name: "title",
-                label: OpenLayers.i18n("Title"),
-                type: "String"
-            }, {
-                name: "comment",
-                label: OpenLayers.i18n("Comment"),
-                type: "String",
-                useTextArea: true
-            }, {
-                name: "legend",
-                type: "LegendAttributeValue"
-            }] : [];
+        this.encodeLayer = this.encodeLayer === undefined ?
+            encodeLayer : this.encodeLayer;
+        this.encodeExternalLayer = this.encodeExternalLayer === undefined ?
+            encodeLayer : this.encodeExternalLayer;
+
+        this.additionalAttributes = this.version == 2 ?
+            this.additionalAttributes === undefined ?
+                [{
+                    name: "title",
+                    label: OpenLayers.i18n("Title"),
+                    type: "String"
+                }, {
+                    name: "comment",
+                    label: OpenLayers.i18n("Comment"),
+                    type: "String",
+                    useTextArea: true
+                }, {
+                    name: "legend",
+                    type: "LegendAttributeValue"
+                }] : this.additionalAttributes : [];
```

Comment:
... and I think that good to put `this.additionalAttributes` on a new line to increase the readability.